projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -829,6 +829,10 @@ public Dictionary<string, string> PlaceOfBirth
             }
             set
             {
+                if (value["addressState"] != "" && !CodeExistsInValueSet(value["addressState"], VR.ValueSets.Jurisdiction.Codes))
```

Comment:
This should use String.IsNullOrEmpty(value["addressState"]) instead of != "" to check for null as well.

Should this be an or check? What should the behavior be if value["addressState"] is empty or null?
I think it should be something like this so it returns null if the value is empty or it's not included in the value set- 

`(String.IsNullOrEmpty(value["addressState"]) || !CodeExistsInValueSet(value["addressState"], VR.ValueSets.Jurisdiction.Codes)`